graphql_compiler/tests/integration_tests/test_backends_integration.py

```diff
@@ -150,7 +151,7 @@ def compile_and_run_query(
         if backend_name in SQL_BACKENDS:
             engine = cls.sql_backend_name_to_engine[backend_name]  # type: ignore
             results, output_metadata = compile_and_run_sql_query(
-                cls.sql_schema_info, graphql_query, parameters, engine  # type: ignore
+                cls.sql_schema_info[backend_name], graphql_query, parameters, engine  # type: ignore
```

Comment:
Is there something we can do about the `type: ignore` here?

Reply:
The error here is `error: "Type[IntegrationTests]" has no attribute "sql_schema_info". We set up per backend attributes in `conftest.py`. Throughout this file we `# type: ignore` on all instances where we're accessing a backend specific attribute to get pylint to pass. Is there a better way to be doing this?

Reply:
Ah okay, so it's a pytest fixture issue. We don't yet have a better way to do this, other than maybe making the `# type: ignore` specific to the error code we expect here (currently it ignores _all_ errors on this line). Not worth fixing in this PR, probably.